### PR TITLE
docs: add guidance to spectro-k8s-dashboard pack DOC-1442

### PR DESCRIPTION
**Pull request by addetz**
## Describe the Change

This PR updates the Kubernetes dashboard reference page. It also fixes the incorrect `-namespace` flag in the legacy kubernetes dashboard pack additional guidance.

## Changed Pages

💻 [Kubernetes dashboard](https://deploy-preview-4474--docs-spectrocloud.netlify.app/clusters/cluster-management/kubernetes-dashboard/)
💻 [Legacy Kubernetes Dashboard pack](https://deploy-preview-4474--docs-spectrocloud.netlify.app/integrations/packs/?pack=k8s-dashboard&version=2.7.0&parent=2.7.x)

## Jira Tickets

🎫 [DOC-1442](https://spectrocloud.atlassian.net/browse/DOC-1442)

## Backports

Can this PR be backported?

- [x] Yes. _Remember to add the relevant backport labels to your PR._

[DOC-1442]: https://spectrocloud.atlassian.net/browse/DOC-1442?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/docs-content/clusters/cluster-management/kubernetes-dashboard.md [hunk 1]
```diff
@@ -8,7 +8,15 @@ tags: ["clusters", "cluster management", "k"]
 ---
 
 The [Kubernetes Dashboard](https://github.com/kubernetes/dashboard) is a general-purpose, web-based UI for Kubernetes
-clusters. You can use the dashboard to manage the cluster, deploy and manage applications, and troubleshoot issues.
+clusters. The dashboard provides a rich variety of functionality by wrapping around the
+[kubectl](https://kubernetes.io/docs/tasks/tools/) CLI. You can use it to perform the following operations.
```
Comment (caroldelwing) [inline docs/docs-content/clusters/cluster-management/kubernetes-dashboard.md:12]:
```suggestion
[kubectl](https://kubernetes.io/docs/tasks/tools/) CLI. You can use it to perform the following operations:
```

File: docs/docs-content/clusters/cluster-management/kubernetes-dashboard.md [hunk 2]
```diff
@@ -8,7 +8,15 @@ tags: ["clusters", "cluster management", "k"]
...
 
-Use the [Spectro Kubernetes Dashboard](../../integrations/spectro-k8s-dashboard.md) pack to add the Kubernetes dashboard
-to your cluster. The pack documentation page has instructions on how to use the pack.
+- List and manage cluster Kubernetes resources.
```
Comment (caroldelwing) [inline docs/docs-content/clusters/cluster-management/kubernetes-dashboard.md:14]:
```suggestion
- List and manage Kubernetes cluster resources.
```

File: docs/docs-content/integrations/spectro-k8s-dashboard.md [hunk 3]
```diff
@@ -17,6 +17,67 @@ tags: ["packs", "spectro-k8s-dashboard", "monitoring"]
 <Tabs groupId="parent">
 <TabItem label="2.7.x" value="2.7.x">
 
+## Access Kubernetes Dashboard
+
+<!-- prettier-ignore-start -->
+The Kubernetes Dashboard pack automatically installs the <VersionedLink text="Spectro Proxy" url="/integrations/packs/?pack=spectro-proxy"/> pack, which serves as a reverse proxy to expose the Kubernetes dashboard and give you access to the host cluster.
+
+Connect to the cluster remotely. Refer to the [Access Cluster with CLI](../clusters/cluster-management/palette-webctl.md#access-cluster-with-cli) guide for further details on how to connect using [Kubectl](https://kubernetes.io/docs/tasks/tools/).
```
Comment (caroldelwing) [inline docs/docs-content/integrations/spectro-k8s-dashboard.md:25]:
```suggestion
1. To access the dashboard, connect to the cluster remotely. Refer to the [Access Cluster with CLI](../clusters/cluster-management/palette-webctl.md#access-cluster-with-cli) guide for further details on how to connect using [Kubectl](https://kubernetes.io/docs/tasks/tools/). 
```
maybe?
Also, what do you think about including numbered steps :)

File: docs/docs-content/integrations/spectro-k8s-dashboard.md [hunk 4]
```diff
@@ -17,6 +17,67 @@ tags: ["packs", "spectro-k8s-dashboard", "monitoring"]
...
 <TabItem label="2.7.x" value="2.7.x">
 
+## Access Kubernetes Dashboard
+
+<!-- prettier-ignore-start -->
+The Kubernetes Dashboard pack automatically installs the <VersionedLink text="Spectro Proxy" url="/integrations/packs/?pack=spectro-proxy"/> pack, which serves as a reverse proxy to expose the Kubernetes dashboard and give you access to the host cluster.
+
+Connect to the cluster remotely. Refer to the [Access Cluster with CLI](../clusters/cluster-management/palette-webctl.md#access-cluster-with-cli) guide for further details on how to connect using [Kubectl](https://kubernetes.io/docs/tasks/tools/). 
+
+Execute the following command from the terminal window to obtain a bearer token. Make a note of the issued token.
```
Comment (caroldelwing) [inline docs/docs-content/integrations/spectro-k8s-dashboard.md:27]:
```suggestion
2. Execute the following command from the terminal window to obtain a bearer token. Make a note of the issued token.
```

File: docs/docs-content/integrations/spectro-k8s-dashboard.md [hunk 5]
```diff
@@ -17,6 +17,67 @@ tags: ["packs", "spectro-k8s-dashboard", "monitoring"]
...
+
+```bash
+kubectl create token kubernetes-dashboard --namespace kubernetes-dashboard
+```
+
+The following example shows what a token value might look like.
+
+```yaml
```
Comment (caroldelwing) [inline docs/docs-content/integrations/spectro-k8s-dashboard.md:35]:
```suggestion
```yaml hideClipboard
```

File: docs/docs-content/integrations/spectro-k8s-dashboard.md [hunk 6]
```diff
@@ -17,6 +17,67 @@ tags: ["packs", "spectro-k8s-dashboard", "monitoring"]
...
+eyJhbGciOiJSUzI1NiIsImtpZCI6Ilg1bTg3RWM4Y1c3NnhkQ3dXbXNDUXQydVpYQklRUWoxa1BaS0ctVkVTSDQifQ.eyJpc3MiOiJrdWJlcm5ldGVzL3NlcnZpY2VhY2NvdW50Iiwia3ViZXJuZXRlcy5pby9zZXJ2aWNlYWNjb3VudC9uYW1lc3BhY2UiOiJrdWJlcm5ldGVzLWRhc2hib2FyZCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VjcmV0Lm5hbWUiOiJrdWJlcm5ldGVzLWRhc2hib2FyZC10b2tlbi1oNGxuZiIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50Lm5hbWUiOiJrdWJlcm5ldGVzLWRhc2hib2FyZCIsImt1YmVybmV0ZXMuaW8vc2VydmljZWFjY291bnQvc2VydmljZS1hY2NvdW50LnVpZCI6IjAwZTcyOWYxLTY2MzgtNGU2OC04ZGY1LWFmYTJlMmUzODA5NSIsInN1YiI6InN5c3RlbTpzZXJ2aWNlYWNjb3VudDprdWJlcm5ldGVzLWRhc2hib2FyZDprdWJlcm5ldGVzLWRhc2hib2FyZCJ9.JU4GOJNjGpkHabUyxBt_2rvtXNjpR3w238BF2oMCQUNf_ZkUGSMeAAgIKxbAuk62dtJNDaRh5yAZ9J5KthMcU6k4qVmodUOJdlvigBVNjTDEhPM-sqJus62HMtwjpvm0CX-aP_A_BqHs2yJ3OgXSX0uHmkUO1FMoZSVaRpOvx7f5bPswxd87L3npuZt4p-NJIX32-DGjBnxdANAHcWil3YHIUbDgQIdjDfN6stGU_JByvzfCJpNCWWDinr772W7iZ3uA28F8uGS0ZMd1E5e1moEFBY8BM015Qxg2Y_k7lmv9S8GMkBJyTiJNiqnwLwfsiE1ycE4Tgq_vuQfFToIMNw
+```
+
+Issue the following command to forward connections from the deployed Kubernetes Dashboard to your local port 8080.
```
Comment (caroldelwing) [inline docs/docs-content/integrations/spectro-k8s-dashboard.md:39]:
```suggestion
3. Issue the following command to forward connections from the deployed Kubernetes Dashboard to your local port 8080.
```

File: docs/docs-content/integrations/spectro-k8s-dashboard.md [hunk 7]
```diff
@@ -17,6 +17,67 @@ tags: ["packs", "spectro-k8s-dashboard", "monitoring"]
...
+<!-- prettier-ignore-end -->
+
+```bash
+kubectl port-forward --namespace kubernetes-dashboard service/kubernetes-dashboard 8080:443
+```
+
+Navigate to `https://localhost:8080` in a browser of your choice to access Kubernetes Dashboard. The Dashboard login
```
Comment (caroldelwing) [inline docs/docs-content/integrations/spectro-k8s-dashboard.md:46]:
```suggestion
4. Navigate to `https://localhost:8080` in a browser of your choice to access the Kubernetes Dashboard. The dashboard login
```

File: docs/docs-content/integrations/spectro-k8s-dashboard.md [hunk 8]
```diff
@@ -17,6 +17,67 @@ tags: ["packs", "spectro-k8s-dashboard", "monitoring"]
...
+page appears.
+
+![Kubernetes dashboard login page](/integrations_spectro-k8s-dashboard_login-page.webp)
+
+Ensure that **Token** is selected. Then, paste the token value you issued earlier into the token field. Click **Sign
```
Comment (caroldelwing) [inline docs/docs-content/integrations/spectro-k8s-dashboard.md:51]:
```suggestion
5. Ensure that **Token** is selected. Then, paste the token value you issued earlier into the token field. Click **Sign
```

File: docs/docs-content/integrations/spectro-k8s-dashboard.md [hunk 9]
```diff
@@ -17,6 +17,67 @@ tags: ["packs", "spectro-k8s-dashboard", "monitoring"]
...
+in**. The Kubernetes Dashboard appears.
+
+## Configure Ingress
+
+Use the following steps to configure ingress for the Kubernetes Dashboard pack.
```
Comment (caroldelwing) [inline docs/docs-content/integrations/spectro-k8s-dashboard.md:56]: Should we maybe explain that the user needs to change these parameters in the values file?

**Review by caroldelwing — APPROVED** [review]
Approved with tiny suggestions :rocket:

File: docs/docs-content/integrations/spectro-k8s-dashboard.md (hunk 9 above)
Comment (addetz) [reply docs/docs-content/integrations/spectro-k8s-dashboard.md:61]: Added this now 👍

**Pull request merged**